cpp/include/cudf/fixed_point/fixed_point.hpp

```diff
@@ -515,6 +515,19 @@ class fixed_point {
     Rep const value = detail::shift<Rep, Rad>(_value, scale_type{scale - _scale});
     return fixed_point<Rep, Rad>{scaled_integer<Rep>{value, scale}};
   }
+
+  /**
+   * @brief Returns a string representation of the fixed_point value.
+   */
+  std::string print() const
```

Comment:
Looks like the function is not used for printing, so `operator<<` is not needed here.

My 2ct: Since this is a cast to `std::string`, I would prefer an actual (explicit) cast operator, instead of `to_string` as @harrism suggested. Either that, or "do as int does" and make `to_string` a free function.

Reply:
This is a debug thing I put in for me, but thought it could be useful for others so I left it. I'm not sure if it should live here or somewhere in the test code, but it made sense here for me. Changing it to `to_string` because that is actually what is happening.

Reply:
Agree with @vuule, good idea on the cast operator.